src/codemodder/codemods/transformations/remove_empty_string_concatenation.py

```diff
@@ -1,12 +1,29 @@
+from typing import Union
 import libcst as cst
-from libcst import CSTTransformer
+from libcst import CSTTransformer, RemovalSentinel, SimpleString
 
 
 class RemoveEmptyStringConcatenation(CSTTransformer):
     """
     Removes concatenation with empty strings (e.g. "hello " + "") or "hello" ""
     """
 
+    def leave_FormattedStringExpression(
+        self,
+        original_node: cst.FormattedStringExpression,
+        updated_node: cst.FormattedStringExpression,
+    ) -> Union[
+        cst.BaseFormattedStringContent,
+        cst.FlattenSentinel[cst.BaseFormattedStringContent],
+        RemovalSentinel,
+    ]:
+        expr = original_node.expression
+        match expr:
+            case SimpleString():  # type: ignore
```

Comment:
is it possible to do `case Simplestring() and expr.raw_value == ...` in one line? This is one case when the multiple indent is kinda exsessive and just having an if statement seems cleaner than using match.

Reply:
Good call on this. I've been writing those type checks on autopilot, there is quite a few of them for this codemod.